app/helpers/entreprise_helper.rb

```diff
@@ -1,6 +1,6 @@
 module EntrepriseHelper
   def raison_sociale_or_name(entreprise)
-    entreprise.raison_sociale.blank? ? entreprise.nom + ' ' + entreprise.prenom : entreprise.raison_sociale
+    entreprise.raison_sociale.blank? ? "#{entreprise.nom} #{entreprise.prenom}" : entreprise.raison_sociale
```

Comment:
to fixup ?

Reply:
J'ai préféré séparer, car dans le commit précédent je veux montrer que j'ai rien changé aux méthodes, juste copié dans un Helper.